var/spack/repos/builtin/packages/henson/package.py

```diff
@@ -0,0 +1,39 @@
+# Copyright 2013-2018 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Henson(CMakePackage):
+    """Cooperative multitasking for in situ processing."""
+
+    homepage = "https://github.com/mrzv/henson"
+    url      = "https://github.com/mrzv/henson"
+    git      = "https://github.com/mrzv/henson.git"
+
+    version('master', branch='master')
+
+    depends_on('mpi')
+
+    variant('python', default=False, description= 'Build Python bindings')
+    extends('python', when='+python')
+    variant('mpi-wrappers', default=True, description= 'Build MPI wrappers (PMPI)')
+
+    conflicts('^openmpi', when='+mpi-wrappers')
```

Comment:
do you want to add a comment which MPI flavors are supported if the spack-default (OpenMPI) is not?

Reply:
MPICH works fine in all configurations. OpenMPI is supported, just not with mpi-wrappers=True.  What's the right solution here? Turn mpi-wrappers off, by default? Is there a way to express default preference for MPICH?

Reply:
I guess since OpenMPI is by default one it will be easier for users to default the `mpi-wrappers` variant to `False` indeed.
> Is there a way to express default preference for MPICH?

Pew, let me ask this in Slack, I don't know.

Reply:
Doesn't look like one can express this currently while still keeping the virtual package dependency on `mpi`.